src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -99,8 +99,19 @@ const mainSelector = createSelector(
   ({ state }) => state.megaMenu,
   ({ megaMenuData }) => megaMenuData,
   (loggedIn, megaMenu, megaMenuData) => {
+    // Derive the default mega menu links (both auth + unauth).
+    const defaultLinks = [
+      ...megaMenuData,
+      // Add the My VA link to default links.
+      {
+        className: 'my-va-top-nav',
+        href: '/my-va/',
+        title: 'My VA',
+      },
```

Comment:
This menu also renders on subdomains like benefits.va.gov, so the link needs to be absolute or they'll end up at `benefits.va.gov/my-va/` (404)

![image](https://user-images.githubusercontent.com/1915775/112226037-05ccd580-8c04-11eb-8067-4b368311e8c8.png)